### PR TITLE
Add DynamiceSlice backend support

### PR DESCRIPTION
**Pull request by chinhuang007**
Add DynmaicSlice handler to the backend. Also update
Less for version 9.

### CODE REVIEW

**Comment by chinhuang007** [comment]
Currently still work-in-progress.

**Comment by chinhuang007** [comment]
The code is ready for review.

**Comment by tjingrant** [comment]
@ibm-jenkins test this please.

**Comment by tjingrant** [comment]
@ibm-jenkins test this please.

File: onnx_tf/handlers/backend/dynamic_slice.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@onnx_op("DynamicSlice")
+@tf_func(tf.slice)
+class DynamicSlice(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    input_tensor = tensor_dict[node.inputs[0]]
+    starts = tf.cast(tensor_dict[node.inputs[1]], tf.int64)
+    ends = tf.cast(tensor_dict[node.inputs[2]], tf.int64)
```
Comment (fumihwh) [inline onnx_tf/handlers/backend/dynamic_slice.py]: Why do we need `cast` here? ONNX ensures `starts` and `ends` are `int32` or `int64` which is accepted in tf.

File: onnx_tf/handlers/backend/dynamic_slice.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    l = list(range(starts.shape[0]))
+    axes = tf.cast(tensor_dict[node.inputs[3]], tf.int64) if len(
+        node.inputs) == 4 else tf.constant(l, tf.int64)
```
Comment (fumihwh) [inline onnx_tf/handlers/backend/dynamic_slice.py]: Also `axes`. `starts`, `ends` and `axes` is ensured the same dtype.

**Comment by fumihwh** [comment]
LGTM!

**Comment by fumihwh** [comment]
But seems CI for release version broken....

**Comment by chinhuang007** [comment]
The CI regression failed probably because the new operator is missing in older onnx builds. So I added a line to check the onnx opset version before running the test.

File: onnx_tf/handlers/backend/dynamic_slice.py (hunk 2 above)
Comment (chinhuang007) [reply onnx_tf/handlers/backend/dynamic_slice.py]: Thanks for the comments. I wasn't sure if axes, starts, and ends are ensure the same type. Now I know, so I just removed some casts.

**Comment by tjingrant** [comment]
Jenkins is happy: http://wb-power-host-8.watson.ibm.com:8080/job/onnx-tensorflow/74

**Pull request merged**